qa/rolling-upgrade/src/javaRestTest/java/org/elasticsearch/upgrades/IndexingIT.java

```diff
@@ -410,19 +409,22 @@ private void assertCount(String index, int count) throws IOException {
         );
     }
 
-    private Version minNodeVersion() throws IOException {
+    private boolean nodeSupportBulkApi(Map<?, ?> nodeInfo) {
+        // TODO[lor]: replace this check with a (historical) feature check ("supports bulk requests")
+        var versionString = nodeInfo.get("version").toString();
+        var version = Version.fromString(versionString.replace("-SNAPSHOT", ""));
```

Comment:
Similar to my comment above, why would we expect the snapshot suffix? Seems like version there should be unqualified, since we have the isSnapshot flag.

Reply:
After https://github.com/elastic/elasticsearch/pull/100868 the replace is not needed anymore, removed.